database/mssql/scripts/dbscripts/PSP_PIMS_S31_01/Alter Down/87_DML_PIMS_PPH_STATUS_TYPE_Edit.sql

```diff
@@ -0,0 +1,26 @@
+/* -----------------------------------------------------------------------------
+Update the PIMS_PPH_STATUS_TYPE table.
+. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 
+Author        Date         Comment
+------------  -----------  -----------------------------------------------------
+Doug Filteau  2022-Apr-12  Initial version
+----------------------------------------------------------------------------- */
+
+BEGIN TRANSACTION;
+
+DECLARE @CurrCd NVARCHAR(20)
+SET @CurrCd = N'COMBO';
+
+SELECT PPH_STATUS_TYPE_CODE
+FROM   PIMS_PPH_STATUS_TYPE
+WHERE  PPH_STATUS_TYPE_CODE = @CurrCd;
+
+IF @@ROWCOUNT = 1
+  BEGIN
+  UPDATE PIMS_PPH_STATUS_TYPE 
+  SET    DESCRIPTION                = N'Combination (PPH & Non-PPH)'
```

Comment:
Should the alter down revert the change? This one looks the same as the Alter up

Reply:
The scripts will repair the problem, no matter what, so I put them in both.